packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -38,6 +38,22 @@ import 'theme.dart';
 ///    integration.
 ///  * [InputDecorator], which shows the labels and other visual elements that
 ///    surround the actual text editing widget.
+///
+/// ## Sample code
+///
+/// ```dart
+/// new TextFormField(
+///   decoration: const InputDecoration(
+///     icon: const Icon(Icons.person),
+///     hintText: 'What do people call you?',
+///     labelText: 'Name *',
+///   ),
+///   onSaved: (String value) {},
```

Comment:
The empty callback seems odd. Maybe describe in a comment what you would do in here?

Reply:
Ok, I added a bit of an explanation which I think should make the example a bit more clear!